### PR TITLE
remove defunct Node.js Foundation Top-Level Projects

### PR DESCRIPTION
**Pull request by Trott**
The Node.js Foundation does not exist anymore. The TSC is not involved
in the process of onboarding other projects. Express and libuv are
currently At-Large Projects with OpenJS Foundation. Remove out-of-date
material about top level projects.

### CODE REVIEW

File: BasePolicies/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,27 +0,0 @@
-## Working Group Code of Conduct Template
```
Comment (nschonni) [inline BasePolicies/CODE_OF_CONDUCT.md:1]: Are these "BasePolicy" files in the nodejs/admin or somewhere else now?

Comment (Trott) [reply BasePolicies/CODE_OF_CONDUCT.md:1]: Bootstrap governance docs are at https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md#bootstrap-governance

Comment (nschonni) [reply BasePolicies/CODE_OF_CONDUCT.md:1]: Fair enough. I guess the `Application/*` files are replaced by https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md#current-working-groups sections as well

**Review by cjihrig — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Review by targos — APPROVED** [review]

**Review by MylesBorins — APPROVED** [review]
LGTM

**Review by BethGriggs — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**